src/osd/scrubber/pg_scrubber.h

```diff
@@ -721,10 +721,13 @@ class PgScrubber : public ScrubPgIF,
 
   int num_digest_updates_pending{0};
   hobject_t m_start, m_end;  ///< note: half-closed: [start,end)
+  int m_num_objects_current_chunk{0};
```

Comment:
This member is used to communicate a number of objects from PgScrubber::select_range() to PgScrubber::select_Range_n_notify().  Instead, modify select_range() to return the number of objects in the chunk (with 0 indicating that it's blocked).  Don't introduce a member  when a return value would suffice.